apps/jupyter/extensions/notebook-runner/src/user-interface.ts

```diff
@@ -56,3 +61,29 @@ export const simplifyUserInterface = async (
 
   hideDisallowedWidgets(mode, app);
 };
+
+/**
+ * A function to hide grading folders in the file browser.
+ * A bit hacky because there is no public API to filter items.
+ * @param fileBrowser The file browser instance
+ */
+const hideGradingFolders = (fileBrowser: FileBrowser): void => {
+  let items: Contents.IModel[] = fileBrowser.model["_items"];
+
+  /**
+   * Override the items getter to filter the desired folder.
+   */
+  Object.defineProperty(fileBrowser.model, "_items", {
+    get() {
+      return items.filter(
+        (item) =>
+          !(item.type === "directory" && item.name.startsWith("grading_")),
```

Comment:
I would very explicit exclude ONLY the two folders that we discussed with Laila.
This is over-reaching and creating potentially more problems down the line. 

If possible, put those two folders in a constants file referenced by the documentation.